src/main/java/com/github/manolo8/darkbot/extensions/plugins/PluginClassLoader.java

```diff
@@ -36,6 +36,13 @@ private static Predicate<String> toMatcher(String pattern) {
         super(urls);
     }
 
+    @SuppressWarnings("unused")
+    public Class<?> loadClass(String name, byte[] bytes) throws ClassNotFoundException {
+        if (PROTECTED.stream().anyMatch(p -> p.test(name)))
+            throw new ClassNotFoundException(name + " is a protected class");
+        return super.defineClass(name, bytes, 0 ,bytes.length);
```

Comment:
```suggestion
        return super.loadClass(name, bytes);
```
I'd rather not skip steps, and still call exactly the same super method even if that is an instant redirection to defineClass

Reply:
loadClass don't have this signature, so I renamed new method (same to super method)
This change help exclude creating child classloader in plugin code for loading generated in-flight bytecode.

Reply:
Plugins should not be generating in-flight bytecode, that sounds totally out of the scope of what plugins should be doing, and if this is not an override but a new method i'd want it not to be added at all unless you can give me a good reason for this

Reply:
This still needs to be reverted for this to be merged